src/components/BBoxEntry.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <b-form-group>
+    <b-form-row>
+      <b-col>
+        <b-form-group label="x_min" label-for="x_min">
```

Comment:
Not sure how this is usually done in forms, but x_min seems a little too technical as a user readable label.

Reply:
I asked around and didn't find a clear answer. I could do something like Lat max/min and Lon max/min.  Would that be clearer to most users?

Reply:
Maybe follow the specification terms? I guess there are names hidden in the API spec documents.

Reply:
looking at the geojson spec, the bounding box uses the terms "southwest" and "northeast" and avoids min/max to avoid confusion when crossing the antimeridian.  I've updated the labels to Southwest latitude/longitude and Northeast latitude/longitude to match that.